server/data/weidu-tp2-ielib.yml

```diff
@@ -3803,6 +3803,9 @@ ielib-resref:
     - name: WIZARD_REMOVE_MAGIC
       detail: resref WIZARD_REMOVE_MAGIC = "SPWI302"
       doc: IElib define
+    - name: WIZARD_DISPEL_MAGIC
+      detail: resref WIZARD_DISPEL_MAGIC = "SPWI302"
```

Comment:
The same resource as Wizard Remove Magic?

Reply:
> The same resource as Wizard Remove Magic?

Yep (see [SPELL.IDS](https://gibberlings3.github.io/iesdp/files/ids/bgee/spell.htm))

Reply:
Irrelevant, this file is generated automatically. This should be handled upstream anyway.